src/UglyToad.PdfPig/Geometry/PdfRectangle.cs

```diff
@@ -33,17 +33,17 @@ public struct PdfRectangle
         /// <summary>
         /// Width of the rectangle.
         /// </summary>
-        public decimal Width { get; }
+        public decimal Width => Right - Left;
 
         /// <summary>
         /// Height of the rectangle.
         /// </summary>
-        public decimal Height { get; }
+        public decimal Height => Top - Bottom;
```

Comment:
While this is an improvement because it prevents repetition of this logic in each constructor it also means these calculations are repeated for every property access. I can't think of a scenario yet where that could cause issues but it's worth bearing in mind though I don't think it needs to be changed back.